### DISCUSSION TITLE
How to display results in a list

### DISCUSSION THREAD

**Discussion by MarkJeronimus** [obfusk/jiten#151] [Q&A]
How do I display the search results in a list format, like other dictionary apps? It always shows results as tiles for me.
Expand/collapse just affects a part of the information in the tile, and when in the collapsed state the tiles are still enormous.
This seems to me like s no-brainer, yet i can't find the option.
compare this with "jisho":
![Screenshot_20230304_235717_Jiten](https://user-images.githubusercontent.com/5273753/222932665-f6835299-df6d-491a-baa7-4a540efd7784.jpg)
![Screenshot_20230304_235859_Jsho](https://user-images.githubusercontent.com/5273753/222932667-07a39dce-0e98-41d3-a7ef-d1d4ff42a4c6.jpg)
The rows in jsjo seem to contain about as much information as the tiles in jiten, but shows 8 times as much on a single screen.

**Comment by obfusk** [comment]
Sorry about the late reply. I have unfortunately not been able to work on Jiten recently.

The Jiten app is essentially running the web server on your device to make it available offline.
So it's really more a mobile website than a native app.

And Jisho's mobile website doesn't exactly seem to be more compact:

![image](https://github.com/obfusk/jiten/assets/1260687/42d4f1e6-6aac-4512-b6f0-11b24c7d02f4)

And tbh the lack of a compact view has never bothered me, but I usually look up Japanese words and only need the first result; nor has anyone requested it before.

However, it does seem like a useful option to have, so I'm happy to add it -- though I don't know when I'll be able to, I have a bit of a backlog :sweat_smile: -- and have created an issue for that: #153.
